__tests__/app.test.js

```diff
@@ -27,4 +27,28 @@ describe("GET /api/topics", () => {
             })
         })
     })
+})
+
+describe("GET /api", () => {
+    test("Should return status 200 and endpoints.json", () => {
+        return request(app).get("/api").expect(200).then((response) => {
+            const endpoints = response.body.newEndpoints;
+            //console.log(endpoints);
+            for (const obj in endpoints) {
+                const value = endpoints[obj];
+                //console.log(value);
+                //console.log(value.description);
```

Comment:
all console.logs and commented out code should be removed before submitting. Always!